api/envoy/admin/v2alpha/BUILD

```diff
@@ -1,7 +1,20 @@
-load("@envoy_api//bazel:api_build_system.bzl", "api_proto_library_internal")
+load("@envoy_api//bazel:api_build_system.bzl", "api_proto_library_internal", "api_proto_package")
 
 licenses(["notice"])  # Apache 2
 
+api_proto_package(
+    name = "v2alpha",
+    srcs = glob(["*.proto"]),
```

Comment:
Can this just be built in to `api_proto_package`? Or at least be the default for `srcs`?

Reply:
I didn't find a way :( glob seems to be context-dependent.

Reply:
Done.